plugin/src/main/java/org/opensearch/ml/task/MLPredictTaskRunner.java

```diff
@@ -105,19 +101,6 @@ public void run(MLPredictionTaskRequest request, TransportService transportServi
         }, e -> listener.onFailure(e)));
     }
 
-    /**
-     * Execute algorithm and return result.
-     * TODO: 1. support backend task run; 2. support dispatch task to remote node
-     * @param request MLExecuteTaskRequest
-     * @param transportService transport service
-     * @param listener Action listener
-     */
-    public void execute(MLExecuteTaskRequest request, TransportService transportService, ActionListener<MLExecuteTaskResponse> listener) {
```

Comment:
Is this a leftover that was implemented in the MLExecuteTaskRunner?

Reply:
Yes, this is some test code, this part should be in `MLExecuteTaskRunner`